### PR TITLE
avatar

### PR DESCRIPTION
**Pull request by barabas007**

### CODE REVIEW

**Comment by barabas007** [comment]
avatar

**Comment by barabas007** [comment]
rubocop

**Review by alxekb — CHANGES_REQUESTED** [review]
Please clean up.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
+
+# Mark the database schema as having been generated.
+db/schema.rb linguist-generated
+
+
+# Mark any vendored files as having been vendored.
+vendor/* linguist-vendored
```
Comment (alxekb) [inline .gitattributes:8]: :eye

File: .github/workflows/backend-ci.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          --health-timeout 5s
+          --health-retries 5       
+
+    steps:
+      - uses: actions/checkout@v1      
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.0.0
+          bundler-cache: true
+      - name: Install dependencies
```
Comment (alxekb) [inline .github/workflows/backend-ci.yml:27]: 🚀

File: .gitignore [hunk 3]
```diff
@@ -1,28 +1,42 @@
...
+
+# Ignore bundler config.
 /.bundle
-/.env
-/coverage/*
-/db/*.sqlite3
-/db/*.sqlite3-journal
+
+# Ignore all logfiles and tempfiles.
 /log/*
```
Comment (alxekb) [inline .gitignore:11]: 😢

File: Rakefile [hunk 4]
```diff
@@ -4,12 +4,3 @@
...
 Rails.application.load_tasks
-task(:default).clear
-task default: [:spec]
-
-if defined? RSpec
-  task(:spec).clear
-  RSpec::Core::RakeTask.new(:spec) do |t|
-    t.verbose = false
-  end
-end
```
Comment (alxekb) [inline Rakefile:15]: 😭

**Review by alxekb — COMMENTED** [review]
Please clean up a little bit more!

**Pull request closed without merging**